spec/models/user_spec.rb

```diff
@@ -1,8 +1,5 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  describe "validations" do
-    it { should validate_presence_of(:uid) }
```

Comment:
Deleted when starting fresh on a new branch, could be re-used to test the new User model's attributes.